src/autopas/LogicHandler.h

```diff
@@ -597,12 +597,14 @@ class LogicHandler {
 
   /**
    * Getter for the mean rebuild frequency.
-   * Helpful for determining the frequency for the dynamic containers
+   * Helpful for determining the frequency for the dynamic containers as well as for determining fast particles by
+   * computing skinPerStep for static container
    * @return value of the mean frequency as double
    */
   [[nodiscard]] double getMeanRebuildFrequency() const {
```

Comment:
This is never used for static containers so is somewhat confusing.

Reply:
done, see [4da4355](https://github.com/AutoPas/AutoPas/pull/821/commits/4da43550d82b376771ada1352a61fc648ff9e0d9)

I was planning to use it for static case but ended up not using it. I have removed it now and added a comment that the function is only used for dynamic containers right now.